api-reference/transaction-path.md

```diff
@@ -4,9 +4,10 @@ Represents a single transaction path.
 
 ## Properties
 
-| Name | Type | Description |
-| :--- | :--- | :--- |
-| `apps` | `App[]` | Get all the apps for the path. |
-| `destination` | `App` | Get the destination of the transactions path. |
-| `transactions` | `TransactionRequest[]` | Transaction request objects, ready to be sent to an Ethereum library. |
-
+| Name                          | Type                     | Description                                                                    |
+| :---------------------------- | :----------------------- | :----------------------------------------------------------------------------- |
+| `apps`                        | `App[]`                  | Get all the apps for the path.                                                 |
+| `transactionsDescribed`       | `TransactionDescribed[]` | List of transactions in the path with their corresponding radspec description. |
```

Comment:
Would it make more sense to move this on `TransactionRequest#description` and `TransactionRequest#descriptionAnnotated`? I don’t think it would be a problem for any Ethereum library, but we could also add a `.export()` that exports a clean transaction request object (only `to` / `from` / `data`), in case a specific context needs it.